Activity 05

Droxinbox opened the pull request:
The Activity 05 is now completed and ready for review.

Droxinbox committed:
Activity 05

The Activity 05 is now completed and ready for review.

erobledoapex commented:
Las instrucciones también piden una captura de pantalla de los resultados en la descripción de la PR

erobledoapex commented on ExercisePart1/src/app/my-first-component/my-first-component.component.spec.ts:
```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```
No son necesarios los unit tests que vienen como boilerplate al usar el CLI, sin embargo, son bienvenidos los unit tests que prueben funciones síncronas que tu hagas

erobledoapex commented on ExercisePart1/src/app/my-first-component/my-first-component.component.ts:
```diff
@@ -0,0 +1,10 @@
+  templateUrl: './my-first-component.component.html',
+  styleUrl: './my-first-component.component.css'
+})
+export class MyFirstComponentComponent {
```
la convención al nombrar componentes es PascalCase del nombre que hayas elegido, sin embargo, no es correcto que el nombre lleve la palabra "component" puesto que eso hace que se repita como en éste caso. Para que el resultado final sea "MyFirstComponent" al usar el cli hay que crear el componente nombrándolo simplemente "my-first" ya que el cli por tratarse de un componente le adjuntará la palabra component tanto al archivo después del punto como al nombre de la clase.

Droxinbox committed:
Activity 05 ver2.0

+Activity 05 has been converted into a component.
+Unnecessary lines of code have been removed.

Droxinbox commented:
![imagen](https://github.com/Droxinbox/AngularPractices/assets/36482757/0a3046ae-0cc0-4fcd-b0b6-a1a988777edf)

Droxinbox commented:
![imagen](https://github.com/Droxinbox/AngularPractices/assets/36482757/bf5a6c68-1f01-44f5-a0de-9ad532d512b0)

Droxinbox committed:
Activity 05 ver 2.1

MyFirstModule name correction

Droxinbox committed:
Merge branch 'main' into Activity_05
